default/stringtables/en.txt

```diff
@@ -1148,16 +1148,16 @@ RULE_HABITABLE_SIZE_DESC
 Value used to adjust for [[encyclopedia POPULATION_TITLE_SHORT_DESC]] changes on a planet of this size.
 
 RULE_THRESHOLD_HUMAN_PLAYER_WIN
-Threshold for human player winners
+Max human player winners
 
 RULE_THRESHOLD_HUMAN_PLAYER_WIN_DESC
-Threshold for count of alive human players for them to be considered winners. Not active for single-player.
+Maximum number of human players that can win together in multiplayer games.
 
 RULE_ONLY_ALLIANCE_WIN
-Only alliance wins
+Only allied players win
 
 RULE_ONLY_ALLIANCE_WIN_DESC
-Only alliance of human players could win. Not active for single-player.
+Disallow multiple human players be winners if someone isn't in alliance with other in multiplayer games.
```

Comment:
This current phrasing seem to be saying something so obvious it is meaningless, in that while it is possible for me to imagine ways to have multiple winners if they were not allied with each other, they are all odd enough that the rule would need to be about ___how___ that could happen, not ___whether___ can or must or must not happen.  From what I thought I understood from the various phrasings you've proposed for this rule, I thought a grammatically correct and concise statement of the rule would be 
```
Single unallied players cannot win.
```
That  was my best guess based one the statements you keep proposing.  It rather begs the question, though,  what would happen if a single unallied player does dominate over everyone else.  I suppose it could be logically consistent to just not award any win points, but that would seem like a rather emotionally unsatisfying outcome and also one that needs to me made really super clear.

Since this made so little sense to me, I have looked through the rest of the PR code, and it looks like what you have actually implemented is in fact not what you seem to keep rephrasing for the rule (nor my restatement above).

What you have implemented looks to me like it could reasonably described as 
```
Allied players win together if the number of humans in their alliance is no larger than the winning-human-survivors-threshold and there are no other human survivors.
```
That phrasing could be be made more succinct if you could really assume that a winning alliance had no non-human players, but even if the current main multiplayer group has an rule against allying with an AI I don't think that this rule can necessarily take that as a given.

Reply:
```
Allied players win together if the number of humans in their alliance is no larger than the winning-human-survivors-threshold and there are no other human survivors.
```
Yep, thanks. That what I mean. AI is simply ignored on this victory type checks. Also other victory types are still available.

Reply:
Fixed text.